### DISCUSSION TITLE
Inquietud Por Permiso

### DISCUSSION THREAD

**Discussion by Enigman24** [Arctosoft/Valv-Android#90] [Q&A]
Hola a todos, Deseo me describan que implicacion y beneficios tiene autorizar DYNAMIC_RECEIVER_NOT_EXPORTED_PERMISSIO en la App , muchas gracias

**Comment by hej2010** [comment]
It is added automatically by Android when you declare `exported="true"` or `exported="false"` in the AndroidManifest for an activity/provider. Valv uses it to allow the app to receive data (files) shared to it from other apps. 
If you disable it you get the error `Permission DYNAMIC_RECEIVER_NOT_EXPORTED_PERMISSION is required by your application to receive broadcasts`.

**Reply by hej2010** [reply]
Read more here: https://stackoverflow.com/a/78816850/7232269

**Reply by Enigman24** [reply]
Muchas gracias por Responder

**Comment by Enigman24** [comment]
Hola, entendido, muchas gracias por responder

Enviado desde Proton Mail para Android

-------- Mensaje original --------
El 20/10/24 4:42 a. m., Rikard Teodorsson  ha escrito:

**Comment by Enigman24** [comment]
Ok, muchas gracias

Enviado desde Proton Mail para Android

-------- Mensaje original --------
El 28/10/24 6:39 a. m., Rikard Teodorsson  ha escrito:
